src/components/dialogs/AddStaffNotationDialog.vue

```diff
@@ -236,10 +246,12 @@ export default class AddStaffNotationDialog extends Mixins(DateMixin, EnumMixin)
       data.filing[this.name] = { ...data.filing[this.name],
         dissolutionType: 'administrative',
         dissolutionDate: this.getCurrentDate,
-        courtOrder: {
+        details: this.notation,
+        ...(this.courtOrderNumber && { courtOrder: {
```

Comment:
it won't be a good idea to use a lot of conditions inside spread operator better to put in the next line with the condition .